docs/source/demos/text_input.ipynb

```diff
@@ -237,6 +237,94 @@
     "As you can see, this model performs relatively well on this dataset, even on unseen data."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What does the Text Featurization Component do?\n",
+    "\n",
+    "Machine learning models cannot handle non-numeric data. Any text must be broken down into numeric features that provide useful information about that text. The Text Featurization Component starts by normalizing your text: it removes any punctuation and other non-alphanumeric characters, and converts any capital letters to lowercase. From there, it passes the text into featuretools' `nlp_primitives` dfs search, resulting in several informative features that replace the original column in your dataset: Diversity Score, Mean Characters per Word, Polarity Score, and LSA (Latent Semantic Analysis).\n",
```

Comment:
would be cool to link to featuretools here where you mention featuretools

Reply:
nit: "The Text Featurization Component starts by normalizing your text: it removes any punctuation and other non-alphanumeric characters, and converts any capital letters to lowercase." ---> "The Text Featurization component first normalizes your text by removing any punctuation and other non-alphanumeric characters, and converts any capital letters to lowercase."